src/constants/ballEvolutions.ts

```diff
@@ -64,6 +65,39 @@ export const basicEvolutions: [StarterBalls, StarterBalls, Balls][] = [
   ['poison', 'wind', 'noxious'],
 ];
 
+export function getEvolutionRecipes(ball: Balls): string | null {
+  let matches = basicEvolutions
+    .filter(([, , result]) => result === ball)
+    .map(([a, b]) => `${ballInformation[a].name} x ${ballInformation[b].name}`);
+
+  if (matches.length === 0) return null;
+
+  // Condense Laser (Horizontal) and Laser (Vertical) recipes into Laser (Either)
+  const isLaser = (s: string) =>
+    /Laser\s*\((Horizontal|Vertical)\)/i.test(s);
+
+  const normalizeLaser = (s: string) =>
+    s.replace(/Laser\s*\((Horizontal|Vertical)\)/gi, "Laser (Either)");
+
+  const condensed: string[] = [];
+
+  for (const m of matches) {
+    const [left, right] = m.split(" x ");
+
+    if (isLaser(left) && isLaser(right)) {
+      condensed.push(m);
+      continue;
+    }
+
+    const normLeft = isLaser(left) ? normalizeLaser(left) : left;
+    const normRight = isLaser(right) ? normalizeLaser(right) : right;
+
+    condensed.push(`${normLeft} x ${normRight}`);
+  }
```

Comment:
I'm a stickler for naming variables that lean towards having more context rather than less. For example if I look below, what does `m` mean? And what does `left` or `right` mean? I find it more valuable to rename it like so:
```suggestion
  for (const match of matches) {
    const [leftBall, rightBall] = match.split(" x ");

    if (isLaser(leftBall) && isLaser(rightBall)) {
      condensed.push(match);
      continue;
    }

    const normLeftBall = isLaser(leftBall) ? normalizeLaser(leftBall) : leftBall;
    const normRightBall = isLaser(rightBall) ? normalizeLaser(rightBall) : rightBall;

    condensed.push(`${normLeftBall} x ${normRightBall}`);
  }
```